showpass/Showpass-E2E-tests
# Added verifyToggleLabelsOnNotificationsPage()

**Description:**
This spec will be verifying the labels' text next to the notification toggles on the ‘Notifications’ page for consistency.

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-23

**How to test:**
Run the following test suite and ensure all tests pass.
npx cypress run --spec "cypress/e2e/user-settings/profile.cy.js"

**Test execution results:**

<img width="1717" alt="all-tests-passed" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/36658e82-39d9-4093-9165-62103adac4d9">

<img width="1162" alt="Screenshot 2023-07-14 at 5 08 52 PM" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/46a2b452-7973-4108-b748-62c1fbaa3a94">
